reana-ui/src/pages/profile/components/Quota.js

```diff
@@ -55,9 +57,52 @@ export default function Quota() {
     );
   }
   return (
-    <Grid columns={2}>
-      {renderPieChart("CPU", quota.cpu)}
-      {renderPieChart("HDD", quota.disk)}
-    </Grid>
+    <>
+      <Grid columns={2}>
+        {renderPieChart("CPU", quota.cpu)}
+        {renderPieChart("Disk", quota.disk)}
+      </Grid>
+      <div className={styles["quota-info"]}>
+        {getQuotaNotifications(quota, config.adminEmail).map(
+          (notification, index) => (
+            <div key={index}>{notification}</div>
+          )
+        )}
+      </div>
+    </>
   );
 }
+
+function getQuotaNotificationInfo(quotaType, exceededLimit, adminEmail) {
+  if (quotaType === "disk" && exceededLimit)
+    return "Please delete unnecessary workflows.";
+  if (quotaType === "disk" && !exceededLimit)
+    return "Please consider liberating some disk space by deleting unnecessary workflows.";
+  if (quotaType === "cpu" && exceededLimit)
+    return (
+      <span>
+        Please contact
+        <a href={`mailto:${adminEmail}`}> REANA administrators</a> for quota
+        limit increase.
+      </span>
+    );
+}
+
+export function getQuotaNotifications(quota, adminEmail, hideLink = true) {
```

Comment:
I think it's OK to keep this functionality there, since it's very related and directly used in `Quota` component. Placing these functions in `utils.py` might only pollute the file, since it will be used only by `Quota` and `NotificationBell` (which later use additional sources to collect other notifications). We might also create a separate quota utils file, but I don't think it's needed. Please share what do you think